src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -28,18 +28,16 @@
  */
 class WebProfilerExtension extends ProfilerExtension
 {
-    private HtmlDumper $dumper;
-
     /**
      * @var resource
      */
     private $output;
 
     private int $stackLevel = 0;
 
-    public function __construct(?HtmlDumper $dumper = null)
-    {
-        $this->dumper = $dumper ?? new HtmlDumper();
+    public function __construct(
+        private ?HtmlDumper $dumper = new HtmlDumper(),
```

Comment:
Please revert. The property wasn't nullable before and you break constructor calls where `null` is passed explicitly.